### DISCUSSION TITLE
Silent aimbot doesn´t work.

### DISCUSSION THREAD

**Discussion by gun3kter** [danielkrupinski/Osiris#2766] [Q&A]
Silent aimbot doesn´t work.

**Comment by dragonfgamer6665** [comment]
[cfg.zip](https://github.com/danielkrupinski/Osiris/files/6314729/cfg.zip)
this is my legit cfg it has 
a little silent aimbot 
backtrack 
you can use triggerbot just need to blind the key
skin changer on 
revel rank on
